docs/DataProviderWriting.md

```diff
@@ -273,12 +273,11 @@ export default {
 };
 ```
 
-## Example Implementation
+## Example REST Implementation
 
 Let's say that you want to map the react-admin requests to a REST backend exposing the following API:
 
-
-### getList
+**getList**
```

Comment:
because when typing 'getList' in the search, this section returns, and because it appears in the side navigation for nothing (the important part is the implementation)